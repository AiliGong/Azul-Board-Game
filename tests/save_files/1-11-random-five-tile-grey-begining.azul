#myformat
3
5
1
a
b
LULRBRUURRYLRYLLYBUBUYUYYURYLYRBULLBBBUYRYRLLLUYRYYBLLYRBLRBUYUURRBRUBLUURBBBRBLYYBYULLUBRLRUYBBLYUR
