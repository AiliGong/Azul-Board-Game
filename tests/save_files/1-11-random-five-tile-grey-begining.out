Factories:
0: F
1: L U L R
2: B R U U
3: R R Y L
4: R Y L L
5: Y B U B

Score for a: 0
Mosaic for a:
1:         . || . . . . . 
2:       . . || . . . . . 
3:     . . . || . . . . . 
4:   . . . . || . . . . . 
5: . . . . . || . . . . . 
broken:

Score for b: 0
Mosaic for b:
1:         . || . . . . . 
2:       . . || . . . . . 
3:     . . . || . . . . . 
4:   . . . . || . . . . . 
5: . . . . . || . . . . . 
broken:
